demos/https/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # AFR HTTPS demo
 afr_demo_module(https)
 
-afr_set_demo_metadata(ID "HTTPS_DEMO")
+afr_set_demo_metadata(ID "HTTPS_S3_DEMO")
```

Comment:
I am concerned with this naming. I dont want the customers to even accidentally think that our http library can work only with S3.

Reply:
I named it it this because when we support upload next, I also want it it to be encompassed under this same demo. Perhaps we should be rid of the S3 part? I mean the MQTT demo is just called MQTT_DEMO even thought it connects to AWS IoT, but it can connect to any MQTT server.